cfrontend/C2C.ml

```diff
@@ -524,6 +524,11 @@ let convertField env f =
   (intern_string f.fld_name, convertTyp env f.fld_typ)
 
 let convertCompositedef env su id attr members =
+  let t = match su with C.Struct -> 
+    let layout = Cutil.struct_layout env members in
+    List.iter (fun (a,b) -> Debug.set_member_offset id a b) layout;
+    TStruct (id,attr) | C.Union -> TUnion (id,attr) in
```

Comment:
Very hard to read because of the lack of indentation for the two cases (Struct/Union).